fieldservice_vehicle_stock/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Field Service Vehicles - Stock',
     'summary': 'Track inventory of Field Service vehicles',
-    'version': '12.0.1.2.2',
+    'version': '12.0.2.0.0',
```

Comment:
Don't do this, it will be doe my the merge bot.
You can add a comment to make sure maintainers use the `major` bump

Reply:
@dreispt He needs to do this to determine the name of the migration directory.

Reply:
@dreispt @max3903 I just reverted the version, I am going to bump it back